modules/ROOT/pages/producing-ign.adoc

```diff
@@ -60,6 +60,40 @@ An easy way to use `fcct` is to run it in a container with `podman`:
 +
 . Use the `example.ign` file to xref:getting-started.adoc[boot FCOS].
 
+The following examples show how to retrieve an Ignition file from a remote source. They are both set to replace the current configuration with a remote Ignition file.
+
+=== Downloading FCCT
```

Comment:
Copy-paste error?

Also, I'm not sure this section really belongs here.  Maybe we should have a separate page explaining config merge and replace?  That functionality seems to trip people up, so it'd be good to explain it more thoroughly.

Reply:
Moved to a separate page. Those are replace examples. I'll add merge after the addressing the outstanding issues.